Source/WebCore/css/CSSCounterStyleDescriptors.cpp

```diff
@@ -67,6 +67,23 @@ static String symbolToString(const CSSValue* value)
     return primitiveValue.stringValue();
 }
 
+static CSSCounterStyleDescriptors::AdditiveSymbols translateAdditiveSymbolsFromStyleProperties(const StyleProperties& properties)
+{
+    auto additiveSymbols = properties.getPropertyCSSValue(CSSPropertyAdditiveSymbols).get();
+    auto& list = downcast<CSSValueList>(*additiveSymbols);
```

Comment:
Do we need a null check here? Didn't check, but this _might_ be why the patch is causing crashes

Reply:
That’s a good point. And I see another problem here. We should not store the result of `get()` in a local variable if the thing we are calling it on is not in one. We need to store the result of `getPropertyCSSValue` in a local.

Reply:
I agree it makes more sense to store the `RefPtr` and then just pass `.get()` when downcasting but the other translation functions follow the same pattern with no problems (even when the rendering path is activated by the other patch).